src/js/mag-jquery.js

```diff
@@ -19,7 +19,7 @@
   if (typeof define === 'function' && define.amd) {
     define(['./mag', './mag-analytics', 'jquery', 'hammerjs', 'prevent-ghost-click', 'jquery-bridget'],
       function (mag, MagnificentAnalytics, $, Hammer) {
-        return (root[name] = factory(mag, MagnificentAnalytics, $, Hammer, PreventGhostClick));
+        return (root[name] = factory(mag, MagnificentAnalytics, jQuery, Hammer, PreventGhostClick));
```

Comment:
@jeffam You either need to revert this line, or also change line 21 above where we pass in `$`.